frontend/src/pages/pipelines/GlobalPipelinesRoutes.tsx

```diff
@@ -13,80 +13,112 @@ import CreateRunPage from '~/concepts/pipelines/content/createRun/CreateRunPage'
+                redirectPath={(namespace) => `/pipelines/${namespace}`}
+              />
+            }
           />
-        }
-      />
-      <Route
-        path="pipelineRunJob/view/:pipelineRunJobId"
-        element={
-          <GlobalPipelineCoreDetails
-            BreadcrumbDetailsComponent={PipelineRunJobDetails}
-            pageName="Pipelines"
-            redirectPath={(namespace) => `/pipelines/${namespace}`}
+          <Route
+            path="pipelineRunJob/view/:pipelineRunJobId"
+            element={
+              <GlobalPipelineCoreDetails
+                BreadcrumbDetailsComponent={PipelineRunJobDetails}
+                pageName="Pipelines"
+                redirectPath={(namespace) => `/pipelines/${namespace}`}
+              />
+            }
           />
-        }
-      />
-      <Route
-        path="pipelineRun/create"
-        element={
-          <GlobalPipelineCoreDetails
-            BreadcrumbDetailsComponent={CreateRunPage}
-            pageName="Pipelines"
-            redirectPath={(namespace) => `/pipelines/${namespace}`}
+          <Route
+            path="pipelineRun/create"
+            element={
+              <GlobalPipelineCoreDetails
+                BreadcrumbDetailsComponent={CreateRunPage}
+                pageName="Pipelines"
+                redirectPath={(namespace) => `/pipelines/${namespace}`}
+              />
+            }
           />
-        }
-      />
-      <Route
-        path="pipelineRun/clone/:pipelineRunId"
-        element={
-          <GlobalPipelineCoreDetails
-            BreadcrumbDetailsComponent={CloneRunPage}
-            pageName="Pipelines"
-            redirectPath={(namespace) => `/pipelines/${namespace}`}
+          <Route
+            path="pipelineRun/clone/:pipelineRunId"
+            element={
+              <GlobalPipelineCoreDetails
+                BreadcrumbDetailsComponent={CloneRunPage}
+                pageName="Pipelines"
+                redirectPath={(namespace) => `/pipelines/${namespace}`}
+              />
+            }
           />
-        }
-      />
 
+          <Route path="*" element={<Navigate to="." />} />
+        </Route>
+      ) : (
+        <Route
+          path="/experiments/:namespace?/*"
+          element={
+            <GlobalPipelineCoreLoader
+              title={experimentsPageTitle}
+              description={experimentsPageDescription}
+              getInvalidRedirectPath={(namespace) => `/pipelines/experiments/${namespace}`}
+            />
+          }
+        >
+          <Route index element={<GlobalExperiments tab={ExperimentListTabs.ACTIVE} />} />
+          <Route path="active" element={<GlobalExperiments tab={ExperimentListTabs.ACTIVE} />} />
+          <Route
+            path="archived"
+            element={<GlobalExperiments tab={ExperimentListTabs.ARCHIVED} />}
+          />
+          <Route path="*" element={<Navigate to="." />} />
+        </Route>
+      )}
```

Comment:
I am passing in the tab for each route... which gets sent through the stream of components to
a) set the active tab
b) render the correct empty state

Reply:
That's fine for me -- runs currently use a query param -- but they could be migrated to use a route part.